tools/recetox_xmsannotator/.shed.yml

```diff
@@ -0,0 +1,16 @@
+owner: recetox
+remote_repository_url: "https://github.com/RECETOX/galaxytools/tree/master/tools/recetox-xmsannotator"
+homepage_url: "https://github.com/RECETOX/recetox-xMSannotator"
+categories:
+  - Metabolomics
+repositories:
+  recetox_xmsannotator_simple:
+    description: "Annotate peak table with metabolites"

```

Comment:
Could be made more detailed to "Annotate peak table using simple database matching."

Reply:
```suggestion
    description: "Annotate peak table using simple database matching"

```